modules/electromagnetics/test/tests/kernels/coupled_electrostatics/tests

```diff
@@ -0,0 +1,72 @@
+[Tests]
```

Comment:
Considering that there is significant re-use of the phrase "The system shall be capable of modeling a div-grad problem in H(div) and in...", let's use the `requirement` and `detail` feature of the test specification system to condense these requirement details a bit. See https://github.com/idaholab/moose/blob/master/test/tests/adaptivity/dont-p-refine/tests for an example.

Reply:
Good catch. Done.